tools/pylib/boutdata/restart.py

```diff
@@ -341,7 +329,7 @@ def is_pow2(x):
 
 
 
-def expand(newz, path="data", output="./", informat="nc", outformat=None):
+def resizeZ(newNz, path="data", output="./", informat="nc", outformat=None):
```

Comment:
`resizeZ` is probably a better name than `expand`, but I'm worried this might break existing scripts. Anybody have strong feelings about this renaming?